.github/workflows/launch-quickstart-mongodbatlas.yml

```diff
@@ -0,0 +1,153 @@
+    - uses: actions/checkout@v2.3.1
+      name: pull code
+    - name: Run the atlas-cfn-deploy custom action.
+      uses: ./.github/actions/atlas-cfn-deploy
+      with:
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        AWS_REGION_INPUT: ${{ github.event.inputs.region }}
+        AWS_REGION_SECRET: ${{ secrets.AWS_REGION }}
+  launch-quickstart-mongodbatlas:
+    name: launch-quickstart-mongodbatlas
+    needs: atlas-cfn-deploy
+    runs-on: ubuntu-20.04
+    #runs-on: ubuntu-latest
+    env:
+      STACK_NAME: ${{ github.event.inputs.projectName }}
+      INSTANCE_SIZE: ${{ github.event.inputs.instanceSize }}
+      DATABASE_USERNAME: ${{ github.event.inputs.databaseUsername }}
+      DATABASE_PASSWORD: ${{ github.event.inputs.databasePassword }}
+      ATLAS_PUBLIC_KEY: ${{ secrets.ATLAS_PUBLIC_KEY }}
+      ATLAS_PRIVATE_KEY: ${{ secrets.ATLAS_PRIVATE_KEY }}
+      ATLAS_ORG_ID: ${{ secrets.ATLAS_ORG_ID }}
+      TEMPLATE_FILE: quickstart-mongodbatlas/templates/mongodbatlas-cfn-quickstart.template.json 
+      CAPABLITIES: CAPABILITY_IAM
+      AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY_ID}}
+      AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+      AWS_REGION: ${{ github.event.inputs.region }}
+    steps:
+    - uses: actions/checkout@v2.3.1
+      name: Pull Code
+    - uses: FranzDiebold/github-env-vars-action@v1.2.0
+    - name: Configure Environment Outputs
+      id: vars
+      run: |
+        #echo "HACK - aws=echo" && cp $(which echo) /usr/bin/aws
+        echo ::set-output name=sha_short::$(git rev-parse --short=5 ${{ github.sha }})
+    - name: Configure Environment
+      id: cfg_env
+      run: |
+        echo "NAME=$NAME"
+      env:
+        NAME: ${{ github.event.inputs.projectName }}-${{ steps.vars.outputs.sha_short }}-${{ github.run_number }}
+    - name: Create AWS Secret for Atlas API Key (if needed)
+      id: create_secret
+      env:
+        NAME: ${{ github.event.inputs.projectName }}-${{ steps.vars.outputs.sha_short }}-${{ github.run_number }}
+      run: |
+        cat << EOF > ./secret.json
+        {
+            "AtlasMongoDBPrivateKey": "${ATLAS_PRIVATE_KEY}",
+            "AtlasMongoDBOrgID": "${ATLAS_ORG_ID}",
+            "AtlasMongoDBPublicKey" : "${ATLAS_PUBLIC_KEY}"
+        }
+        EOF
+        cat ./secret.json
+        echo "standby, baking a fresh yummy secret for your MongoDB Atlas API Key..."
+        echo "Provisioning AWS Secret for MongoDB Atlas API Key called 'mongodbatlas/atlas-cfn-quickstart/$NAME"
+        secret_info=$(aws secretsmanager create-secret \
+         --name="mongodbatlas/atlas-cfn-quickstart/$NAME" \
+         --region="${AWS_REGION}" \
+         --secret-string="file://./secret.json" ) 
+        echo "secret_info=$secret_info"
+        echo ::set-output name=secret_info::$(echo $secret_info)
+    - name: Create a new MongoDB Atlas Cloud Native CFN Quickstart stack
```

Comment:
I'd check w/ Jenny (or who she partners you with) to help w/ naming - don't think I've seen us use Cloud Native.

Reply:
updating to ` Create a new MongoDB Atlas CloudFormation Quickstart stack` for now.